librabft-v2/src/record.rs

```diff
@@ -1,12 +1,10 @@
 // Copyright (c) Calibra Research
```

Comment:
Btw we should change that on all files (I will do that later)
```suggestion
// Copyright (c) Novi Research
```